contributor-growth/README.md

```diff
@@ -0,0 +1,71 @@
+# Contributor Growth
+
+The Contributor Growth subproject is interested in assisting CNCF projects with
+sustainably growing their contributor base.
+
+## Goals
+
+* Provide best practices and resources for managing and interacting with
+  contributors to CNCF projects.
+* Provide guidance on a sustainable contributor lifecycle for CNCF Projects to:
+  * Develop a pipeline that brings in new contributors.
+  * Define a contribution ladder with activities that promote contributors to
+    higher levels of responsibility.
+  * Support their maintainers with advice and resources that improve sustainability
+    and plan for transitions from the project.
+    
+
+## Scope
+
+* Templates for common documents like a contributing guide, reviewing guide,
+  maintainers guide, contribution ladder, etc.
+* Case studies of successful projects of various kinds/sizes and how they manage
+  contributions.
+* Collect and share talks related to the subproject's mission, for example
+  building open source communities, or reflections on a particular community's
+  best practices.
+* Document ways to contribute to a project, especially roles outside of the
+  obvious feature enhancement/code contributions.
+* Create spaces for maintainers of CNCF projects to share their own best
+  practices and ask questions.
+* Discuss non-code areas that support contributor growth, such as project
+  management, design, user experience and documentation and how CNCF projects
+  can prioritize work that may not be the maintainers primary skillset.
+* Provide materials that a CNCF project can use to address the TOC [due
+  diligence guidelines] related to project's contributors and community.
+
+[due diligence guidelines]: https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md#project
+
+### Out-of-Scope
+
+The following are examples of things out-of-scope:
+
+* Project governance.
+* Project automation, such as prow.
+* Day to day support for improving the contribution experience for CNCF
+  projects. Some CNCF projects have support for this, for example within
+  Kubernetes, that would be Kubernetes SIG Contributor Experience. Even when
```

Comment:
I'm still a bit confused by this line -- "Even when there is not a relevant CNCF project these activities are out-of-scope for this subproject." Can we clarify this to make sure there's only one interpretation of it? 

Correct me if I'm wrong but I believe we're trying to say that some projects take on the day to day support themselves (as, an example, through the K8s SIG) but in the absence of a project providing its own day to day support, that does not mean the burden falls onto this group. 

If so, how about -- "Day to day support for improving the contribution experience for CNCF projects. Instead, CNCF projects can take on this support themselves (for example, the CNCF project Kubernetes has its own SIG Contributor Experience)."

I'm hoping by stating what _they_ can take on, we eliminate ourselves of that responsibility. Does that make sense? 😅

Reply:
Yes, thank you that _is_ what I am trying to say. Let me update it with your wording. 👍